pkg/provider/provider.go

```diff
@@ -108,7 +112,11 @@ func (p *Provider) Validate(ctx context.Context, r *externaldata.ProviderRequest
 		if err != nil {
```

Comment:
In the future, I think we might want to distinguish between
* an error talking to the bundle registry
* a nonexistent bundle

This way, we wouldn't return `$key_invalid` if there's, for example, a network issue when talking to the bundle registry.